uint32 row
uint32 col
---
string cell_name
geometry_msgs/Pose  pose
bool available